app/assets/javascripts/discourse/app/components/user-stream-item.js

```diff
@@ -1,16 +1,20 @@
 import Component from "@ember/component";
 import { propertyEqual } from "discourse/lib/computed";
+import { computed } from "@ember/object";
 import { actionDescription } from "discourse/widgets/post-small-action";
 
 export default Component.extend({
   classNameBindings: [
     ":user-stream-item",
     ":item", // DEPRECATED: 'item' class
-    "item.hidden",
+    "hidden",
     "item.deleted:deleted",
     "moderatorAction",
   ],
 
+  hidden: computed("item.hidden", function () {
+    return this.get("item.hidden") && !this.currentUser.staff;
```

Comment:
Will this work if the user is anonymous? Normally we check if `currentUser` exists.

Reply:
Good catch!